packages/fast-form-generator-react/src/utilities/form-input.style.ts

```diff
@@ -5,25 +5,16 @@ import {
     localizeSpacing,
```

Comment:
please move the contents of this file to the `form` folder and rename the file to `form.utilities.style.ts`